src/pages/jobs.js

```diff
@@ -0,0 +1,160 @@
+import { useState } from 'react'
+import Form from '../components/Forms/JobForm/Form'
```

Comment:
Please rename JobForm to be consistent with `Form` naming convention and use a consistent import statement.